builder/oracle/oci/builder.go

```diff
@@ -84,7 +84,7 @@ func (b *Builder) Run(ctx context.Context, ui packer.Ui, hook packer.Hook) (pack
 		return nil, rawErr.(error)
 	}
 
-	region, err := b.config.ConfigProvider.Region()
```

Comment:
there's a fair number of places where you made previously exported fields unexported. Did these fields break the HCL2 generation?

Reply:
Yes, the generation wanted to generate those but they are set from processing. It would work to leave them but I _think_ it's more idiomatic to have them private.